test/channels/application_cable/connection_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+class ApplicationCable::ConnectionTest < ActionCable::Connection::TestCase
```

Comment:
open `.gitignore` file and add `test`
then run command `git rm --cached -r test`

Reply:
Done